data/index.js

```diff
@@ -17,18 +17,31 @@ const initialReducer = () => ( {} );
 const store = createStore( initialReducer, {}, flowRight( enhancers ) );
 
 /**
- * Registers a new sub reducer to the global state
+ * Registers a new sub reducer to the global state and returns a Redux-like store object.
  *
- * @param {String} key     Reducer key
- * @param {Object} reducer Reducer function
+ * @param {String}  key     Reducer key
+ * @param {Object}  reducer Reducer function
+ *
+ * @return {Object}         Store Object
  */
 export function registerReducer( key, reducer ) {
 	reducers[ key ] = reducer;
 	store.replaceReducer( combineReducers( reducers ) );
-}
 
-export const subscribe = store.subscribe;
+	return {
+		dispatch: store.dispatch,
+		getState: () => get( store.getState(), key ),
```

Comment:
Is `get` necessary here (or anywhere we're calling `getState()`)?

Reply:
You can introduce the following helper method:
```js
const getState = () => get( store.getState(), key );
```

Reply:
I guess it's not necessary after all